tests/unit/migrations.py

```diff
@@ -151,7 +151,10 @@ def create_bigint_db():
         db.session.add(model.NetworkingAction(type='modify_port',
                                               nic=nic1,
                                               new_network=network1,
-                                              channel='vlan/100'))
+                                              channel='vlan/100',
+                                              status='PENDING',
+                                              uuid='60bcb0f3-2ee0-4efa-b6ce-e8'
+                                              '1596c6386e'))
```

Comment:
I kinda don't like this being split across two lines. I think it will let you do:

```python
db.session.add(model.NetworkingAction(
    type='modify_port',
    ...
    uuid=...))
```